aequilibrae/project/network/osm_builder.py

```diff
@@ -111,14 +120,17 @@ def importing_links(self, node_count):
 
         logger.info("Adding network links")
         self.__emit_all(["text", "Adding network links"])
-        self.__emit_all(["maxValue", len(self.links)])
+        L = len(list(self.links.keys()))
+        self.__emit_all(["maxValue", L])
 
         nodes_to_add = set()
         counter = 0
         mode_codes, not_found_tags = self.modes_per_link_type()
 
         for osm_id, link in self.links.items():
             self.__emit_all(["Value", counter])
+            counter += 1
+            self.__emit_all(["text", f"{counter:,} of {L:,} super links added"])
```

Comment:
maybe redundant because you have error logging, but this is "about to add" or "adding", not added, right?

Reply:
True